.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: Testing
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ 8540-wordpress-implement-github-actions ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  unittest:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
+        php-versions: ['7.0', '7.1', '7.2', '7.3']
+    name: Unittest on ${{ matrix.operating-system }} with PHP version ${{ matrix.php-versions }}
+    steps:
+    - name: Set up MySQL
+      uses: mirromutth/mysql-action@v1.1
+      with:
+        mysql version: '5.7'
+        mysql root password: '123456'
```

Comment:
Is it ok to use a public password here? I do think it is, but is there any documentation on this?

Reply:
we can transfer a secret value from env variable